ci/tasks/smoke-test-darwin.sh

```diff
@@ -1,33 +1,21 @@
-#! /bin/sh
+#! /bin/bash
 
 set -e
 
-: "${VERSION?The pipeline must set this parameter}"
+banner() {
+    echo
+    echo "######################################################################"
+    echo "$*"
+}
 
+: "${VERSION?The pipeline must set this parameter}"
 cd artifacts
-# ITERATE OVER ALL THE DARWIN ARCHIVES
-for MAC_ARCHIVE in $(ls *.zip); do
-    # CREATE A SPECIFIC DIRECTORY FOR THE ARTIFACT TO PREVENT OVERWRITING
-    ARTIFACT_DIRECTORY=$(basename $MAC_ARCHIVE .zip)
-    mkdir $ARTIFACT_DIRECTORY
-    unzip $MAC_ARCHIVE -d $ARTIFACT_DIRECTORY
 
-    echo
-    echo Entering $ARTIFACT_DIRECTORY
-    cd $ARTIFACT_DIRECTORY
+unzip *.zip
```

Comment:
Ah, you don't need the for loop because you're not cross-building anymore, so there's only one zip ?

Reply:
I had to ask @SpectreVert. The loop is a leftover from I think fly_helper, where we were building for 2 macOS archs: amd64 and arm64. Since we are building for arm64, no need to loop.